docs/wiki/01_STUDY_PARAMETERS_SETUP/01_Study_Variables/00_Introduction_to_Study_Variables.md

```diff
@@ -1,15 +1,28 @@
+# Introduction to LORIS Study Variables
 
-# Study Variables Setup
+## Identifiers
 
-### Overview
-This page covers how to set up LORIS with basic parameters for research data. 
+In LORIS, and indeed in all of its instances (CBIG, CCNA, xxx, yyy, etc), the names of participants is anonymized by assigning numeric and alphanumeric ID's to them. In this way, the data of a participant is protected from connection to their name or personal identifying information ("PII"). 
```

Comment:
Preferable to not have specific project names in core LORIS (CCNA, CBIG ,...).

I would also rephrase this a bit in the style "LORIS is nto intended to store PII, Identifiers are used for data linkage"